lib/questions.rb

```diff
@@ -1,22 +1,27 @@
 # keep only the elements that start with an a
 def select_elements_starting_with_a(array)
+  array.select{|string| string[0] == 'a'}
 end
 
 # keep only the elements that start with a vowel
 def select_elements_starting_with_vowel(array)
+  array.select { |string| string[0] == string[/[aeiou]/] }
 end
 
 # remove instances of nil (but NOT false) from an array
 def remove_nils_from_array(array)
+  array.compact
 end
 
 # remove instances of nil AND false from an array
 def remove_nils_and_false_from_array(array)
+  array.select {|string| string != nil && string != false }
```

Comment:
Prefer `!expression.nil?` over `expression != nil`.<br>Space between { and | missing.